.github/workflows/clang_lint/msbuild/clang_tidy_runner.py

```diff
@@ -66,7 +66,11 @@ def run(*kwargs):
     print("==== Running clang-tidy with the following arguments:")
     print(cli)
 
-    proc = subprocess.run(cli)
+    for i in range(0, 9):
```

Comment:
Not a big fan, but yeah, sounds like the best option for now.
You added some rationale in the PR description, can you copy/paste it here as I don’t think we will remember this in few weeks time?

Reply:
Thanks :)

Reply:
Done.